tests/test_verifylib.py

```diff
@@ -395,26 +395,45 @@ def setUp(self):
     self.materials = {
       "foo": {"sha256": self.sha256_foo},
       "foobar": {"sha256": self.sha256_foobar},
+      "sub/foobar": {"sha256": self.sha256_foobar},
       "sub/foo": {"sha256": self.sha256_foo},
-      "sub/foobar": {"sha256": self.sha256_foobar}
-
+      "lib/build/foobar": {"sha256": self.sha256_foobar},
+      "lib/foo": {"sha256": self.sha256_foo},
+      "lib/build/foo": {"sha256": self.sha256_foo},
+      "lib/foobar": {"sha256": self.sha256_foobar},
+      "build/lib": {"sha256": self.sha256_foobar},
+      
     }
     self.products = {
       "bar": {"sha256": self.sha256_bar},
       "barfoo": {"sha256": self.sha256_barfoo},
+      
+      "lib": {"sha256": self.sha256_bar},
+
+      "lib/bar": {"sha256": self.sha256_bar},
+      "lib/barfoo": {"sha256": self.sha256_barfoo},
+      
+      "sub": {"sha256": self.sha256_bar},
       "sub/bar": {"sha256": self.sha256_bar},
       "sub/barfoo": {"sha256": self.sha256_barfoo},
-      }
+
+      "sub/lib/bar": {"sha256": self.sha256_bar},
+      "sub/lib/barfoo": {"sha256": self.sha256_barfoo},
+
+      "build": {"sha256": self.sha256_bar},
+      "build/bar": {"sha256": self.sha256_bar},
+      "build/barfoo": {"sha256": self.sha256_barfoo},
+    }
 
     self.links = {
-        "dest-item": Metablock(signed=Link(
-            name="dest-item",
-            materials=self.materials,
-            products=self.products)),
+        "dest-item": Metablock(signed=Link(name="dest-item",materials=self.materials,products=self.products)),
+        #,"compilation": Metablock(signed=Link(name="compilation",materials=self.materials,products=self.products))
```

Comment:
It's likely safe to get rid of this, since it doesn't seem to be required for the tests.